service/awsconfig/v8/key/key.go

```diff
@@ -35,6 +35,10 @@ const (
 	PolicyNameTemplate = "EC2-K8S-Policy"
 )
 
+const (
+	InstanceIDAnnotation = "aws-operator.giantswarm.io/instance"
```

Comment:
The annotation is applied to the node config specific on AWS. This is to make the reconciliation of existing node configs more reliable and independent from EC2 instance states.